client/js/widgets/button.js

```diff
@@ -78,11 +78,17 @@ export class Button extends Widget {
         }
       }
       if(a.skip) {
```

Comment:
Maybe then also call it `skipif`?

Reply:
I'd prefer `unless` for better readability, but since people are probably already familiar with `skip`, I'd rather keep it as is.

Reply:
Alternatively, we could use `skipunless` as the obvious opposite of `skip`, but it seems a bit lengthy

Reply:
I don't have a personal preference between "skipif" and "unless" but I'm happy with a name change with such an important change to the feature. I think "skipif" is familiar enough that people seeing it will understand that it is the same feature, but different so people will know that the feature has changed in an important way.

Downvote "skipunless".

Reply:
How about `skipIF` and `doIF`?